timezone/src/include/timezone/dialogs.rb

```diff
@@ -954,7 +970,8 @@ def TimezoneDialog(args)
             ret = :again
             timezone = timezone_old
           end
-          Builtins.y2milestone("timezone %1 ret %2", timezone, ret)
+
+          Builtins.y2milestone("timezone %1 ret %2, hwclock %3 -> %4", timezone, ret, hwclock_s_old, @hwclock_s)
```

Comment:
NP: the same here

Reply:
As said above the code is rather in need of some refactoring effort. It's not only the ycpkiller leftovers but also the whole code organization with the internal state distributed everywhere,
unused method arguments, functionless dialog elements, and misleading button labels.

So I'd rather not change the logging interface at this point.

Reply:
Maybe I should add a doc listing the issues I saw.

Reply:
> Maybe I should add a doc listing the issues I saw.

It seems to me a nice idea :)